thrust/thrust/pair.h

```diff
@@ -73,10 +73,49 @@ using tuple_size = ::cuda::std::tuple_size<T>;
  *          provided by <tt>pair::second_type</tt>.
  */
 template <class T, class U>
-using pair = ::cuda::std::pair<T, U>;
-
```

Comment:
I just fell over this again because of a doxygen issue and now wonder, why didn't you just pull `pair` into the `thrust` namespace without creating an alias? Like:
```c++
using _CUDA_VSTD::pair;
```
This works with CTAD and passes the unit tests.